examples/tutorials/tutorial_1_aa_setup.m

```diff
@@ -0,0 +1,98 @@
+% This is a script/function to help set up your user specific paramater xml
+% file.
+%
+% It runs a bare minimum tasklist using OpenNeuro dataset ds000114 (that
+% will be automatically downloaded if you have not done so yet).
+% This isn't a useful analysis; it's intended to check that your
+% paths, parameter files, etc are set up correctly.
+%
+% This file also shows the default function calls that will be used
+% in an all aa run scripts / functions:
+%   aa_ver5
+%   aarecipe
+%   aas_processBIDS % when using BIDS data
+%   aa_doprocessing
+%   aa_close
```

Comment:
Users typically do not run `aa_close` manually, because it is called by `aa_doprocessing`. Users have to run `aa_close` only if aa crashes and they want to restart the analysis.

Reply:
I have removed the reference and call to aa_close, also from the other example scripts that had it. I did leave them in the developer/testscripts, though they are probably also superfluous there?

Reply:
Indeed! I missed the testscripts. :)